vue/components/ui/molecules/table/table.vue

```diff
@@ -1,7 +1,27 @@
 <template>
-    <table class="table" v-bind:class="classes" v-bind:style="style">
+    <table
+        class="table"
+        v-bind:class="{ classes, disableSelection: lastIndexWithShift !== null }"
+        v-bind:style="style"
+    >
+        <global-events
+            v-on:keydown.meta.65.exact.prevent="onCtrlA()"
+            v-on:keydown.ctrl.65.exact.prevent="onCtrlA()"

```

Comment:
When clicking CTRL+the penultimate checkbox and CTRL+last checkbox and then doing SHIFT+up, a weird behaviour happens

When selecting an item with the mouse and then selection other with CTRL, when doing SHIFT+up, it starts from the one clicked from the mouse.